packages/record/src/components/RecordHeader/index.js

```diff
@@ -12,21 +13,43 @@ import {
     RowContainer,
     TagsContainer,
     Body,
+    LabelLoadingShape,
+    DescriptionLoadingShape,
 } from './styled';
 
 export default function RecordHeader(props) {
-    const { className, style, label, description, icon, tags, children, actions } = props;
+    const {
+        className,
+        style,
+        label,
+        description,
+        icon,
+        tags,
+        children,
+        actions,
+        isLoading,
+    } = props;
     return (
         <Container className={className} style={style}>
             <Header>
                 <Details>
                     <RowContainer>
-                        <IconContainer>{icon}</IconContainer>
-                        <Label>{label}</Label>
+                        <RenderIf isTrue={isLoading}>
+                            <LabelLoadingShape />
+                        </RenderIf>
+                        <RenderIf isTrue={!isLoading}>
+                            <IconContainer>{icon}</IconContainer>
```

Comment:
same ^^ here we are creating a dom element when nothing will render if an icon is not passed